internal/handlers/handlers.go

```diff
@@ -397,8 +398,9 @@ func ChannelsHandler(c *fiber.Ctx) error {
 		// Set the Content-Disposition header for file download
 		c.Set("Content-Disposition", "attachment; filename=jiotv_playlist.m3u")
 		c.Set("Content-Type", "application/vnd.apple.mpegurl") // Set the video M3U MIME type
-		c.Response().Header.Set("Cache-Control", "public, max-age=3600")
-		return c.SendString(m3uContent)
+		c.SendStream(strings.NewReader(m3uContent))
+		return nil
+
```

Comment:
What's going on here? Why change?
Keep it as is.

Reply:
this change is required 
otherwise it wont work in webos

Reply:
That's illogical mate.
Care to explain why?

Reply:
the reason to return c.SendStream is for proper error formatting. Incase if something goes wrong, it will return `error`. Otherwise `nil`
I really can't understand how that would be a fix for webos

Reply:
> That's illogical mate. Care to explain why?
If i dont use SendStream, content of m3u will be sent as file all at once 
but webos cannot download file and save it to process it further 
so sending content of file in stream is necessary 
SendStream will send m3u in chunked data as a stream

Reply:
@dheerajv4855 My bad. I misread it as `SendString`